lms/djangoapps/support/static/support/jsx/entitlements/components/Search/Search.jsx

```diff
@@ -24,7 +24,7 @@ class Search extends React.Component {
 
   render() {
     return (
-      <form onSubmit={this.handleSubmit}>
+      <form onSubmit={this.handleSubmit} className="col-md-3 search-form">
```

Comment:
nice.

Reply:
I'm basically a Bootstrap expert. Try not to be too intimidated.

Reply:
FYI [the `classnames`](https://github.com/JedWatson/classnames) library can also combine classes like

```javascript
import classNames from 'classnames';

<form className={classNames('col-md-3', 'search-form')} />
```

This is obviously a trivial case and `classnames` is probably overkill, but I find it's API flexible and powerful, particularly when it comes to conditional classes (again, not relevant here).

Reply:
Also, I'm not even close to being a `Bootstrap` newbie - do you mind explaining why `col-md-3` is appropriate (vs. `col-sm` or `col-lg` or whatever the other bazillion configuration options are)?

Reply:
On all screens of size medium or larger, having the search form and button look inline with one another looks cleaner and adds a the visual cue to indicate their distinct functionality, but on smaller screens it would look to cramped and the search form would be unusably small, so it should take up the full width of smaller screens rather than being limited to just a quarter of the screen.

Reply:
As far as using the classNames API, I agree that it'd be overkill here. I will keep it in mind in the future, though, when using conditional and/or dynamic class names.